### ISSUE TITLE
missing examples/ssd/01_finetune_ssdnet.sh

### ISSUE THREAD

**Issue by lbin** [chaitu2289/ristretto_ssd#1]
Hi @chaitu2289 

There is no file named 01_finetune_ssdnet.sh, could you please upload this file?

Thanks~

**Comment by navneet1083** [comment]
were u able to run 00_quantize_ssdnet.sh ?

**Comment by lbin** [comment]
Yes. There is too much missing in this repo, should do lots of work to make it run.

**Comment by navneet1083** [comment]
yes, the PASCAL VOC is also not downloadable. i ran for the imagenet data-set, wanted to try with SSD model but the prototxt file itself is giving so much errors. Do you have the pascal voc prototxt file for SSD ?

**Comment by chaitu2289** [comment]
@lbin I have uploaded the 01_finetune_ssdnet.sh.  Please let me know what other issues you were facing. I will add steps to the documentation

@navneet1083 I did not understand the issue. Can you please elaborate it ?

**Comment by navneet1083** [comment]
thanks @chaitu2289 , now i am able to download VOC files.

**Comment by navneet1083** [comment]
i am getting following error while try to run - ./examples/ssd/00_quantize_ssdnet.sh 

F0408 13:53:20.977342 11233 insert_splits.cpp:29] Unknown bottom blob 'data' (layer 'conv1_1', bottom index 0)

train.prototxt - https://www.dropbox.com/s/1svwmv0jk051pqq/train.prototxt?dl=0

**Comment by lbin** [comment]
Hi @chaitu2289  I had already run the 01, however, I could compare the codes and results with yours
Thanks
